### DISCUSSION TITLE
Next.js 14,  <Link/ >`scroll={false}` does not seem to work as expected

### DISCUSSION THREAD

**Discussion by borachoidev** [vercel/next.js#72750] [Help]
### Summary

Hi, I'm struggling with page scrolling behavior.
`<Link scroll={false} >` does not seem to work as expected.

```
(App) 
└── (myApp)
    ├── layout.tsx
    ├── page.tsx
    └── (one)
        └── about
            └── page.tsx
      
```

I am using the ` <Link/>` component in `/`  to navigate to the `/about` page, but I do not want the scroll position to be preserved. Despite using `scroll={false}`, the scroll position is still being retained from the previous page.

Is there a way to ensure that the new page starts at the top instead of preserving the scroll position from the previous page?

Thank you.

### Additional information
```
 App router 
 next version :  14.2.13
 ```

**Comment by Choi-Jiwon-38** [comment]
To ensure the new page starts at the top, you should set scroll={true} in the Next.js <Link /> component. By default, setting scroll={true} resets the scroll position when navigating, while scroll={false} retains the previous position.

Try changing it to scroll={true}, and it should work as expected! 👍
